CHANGELOG-2.3.md

```
@@ -0,0 +1,7 @@
+# CHANGELOG FOR `2.3.X`
+
+## v2.3.0 (2026-05-08)
+
+#### Details
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)